workflows/data_pipelines/uai/uai_config.py

```diff
@@ -0,0 +1,34 @@
+from dag_datalake_sirene.config import (
+    MINIO_BASE_URL,
+    DataSourceConfig,
+    DATA_GOUV_BASE_URL,
+    DATAGOUV_URL,
+)
+
+
+UAI_CONFIG = DataSourceConfig(
+    name="uai",
+    tmp_folder=f"{DataSourceConfig.base_tmp_folder}/uai",
+    minio_path="uai",
+    file_name="uai",
+    files_to_download={
+        "mesr": {
+            "url": f"{DATA_GOUV_BASE_URL}bcc3229a-beb2-4077-a8d8-50a065dfbbfa",
+            "resource_id": "bcc3229a-beb2-4077-a8d8-50a065dfbbfa",
+            "destination": f"{DataSourceConfig.base_tmp_folder}/uai/uai-mesr-download.csv",
+        },
+        "menj": {
+            "url": f"{DATAGOUV_URL}/api/1/datasets/5889d03fa3a72974cbf0d5b1",
+            "dataset_id": "5889d03fa3a72974cbf0d5b1",
+            "destination": f"{DataSourceConfig.base_tmp_folder}/uai/uai-menj-download.csv",
+        },
+        "onisep": {
+            "url": f"{DATAGOUV_URL}/api/1/datasets/5fa5e386afdaa6152360f323",
+            "dataset_id": "5fa5e386afdaa6152360f323",
+            "destination": f"{DataSourceConfig.base_tmp_folder}/uai/uai-onisep-download.csv",
+        },
+    },
```

Comment:
I understand the point of adding an extra layer for multi-sourced data. It seems like it adds an extra layer for all other data sources. 

UAI and Colter are the only data sources with multiples urls. I don't know how we should handle those two cases.

Reply:
I think we need a common structure otherwise we will transfer the complexity to where the variable are used like in `DataProcessor.save_date_last_modified()`, `DataProcessor.download_data()` or even `preprocess_data()`.

What if we need to add a second file later on to finess? This would require a refactor of the already implemented file before we can add the second one.

But once the config structure stablises, I feel like we will end up with a lot of redundant settings. Such as the `destination` key which could be derived from the others. We could then challenge the implementation again to reduce the number of hard coded variables.

What do you think?

Reply:
Okay then! We'll cross that bridge when we come to it!